core/src/main/java/org/mskcc/cbio/portal/scripts/ImportExtendedMutationData.java

```diff
@@ -73,7 +73,7 @@
     private Set<String> filteredMutations = new HashSet<String>();
     private Set<String> namespaces = new HashSet<String>();
     private Pattern SEQUENCE_SAMPLES_REGEX = Pattern.compile("^.*sequenced_samples:(.*)$");
-    private final String ASCN_NAMESPACE = "ascn";
+    private final String ASCN_NAMESPACE = "ASCN";
```

Comment:
@averyniceday can see any problem with this viz-a-viz legacy data?  i.e. it was working before with lowercasing, now it won't?

Reply:
No, not really. The manual describes that ASCN should be represented with capitals in the meta_mutations_extended.txt file. The lowercase conversion happens in the backend and ASCN_NAMESPACE matched this converted value. In a previous PR I have removed this conversion to lowercase. This fix will correct the match to the original case present in the meta_mutations_extended.txt file.

TLDR Legacy data is only affected when using lowecase _ascn_ in the metafile. This is not as described in the documentation and I consider this as making use a unsupported feature.